Purchases/Public/RCPurchases.m

```diff
@@ -163,47 +178,22 @@ - (void)addAttributionData:(NSDictionary *)data
     }
 }
 
-- (void)setDelegate:(id<RCPurchasesDelegate>)delegate
-{
-    _delegate = delegate;
-
-    if (delegate != nil) {
-        self.storeKitWrapper.delegate = self;
-        [self.notificationCenter addObserver:self
-                                    selector:@selector(applicationDidBecomeActive:)
-                                        name:APP_DID_BECOME_ACTIVE_NOTIFICATION_NAME object:nil];
-        [self readPurchaserInfoFromCache];
```

Comment:
We were emitting the last cached version of PurchaserInfo in the past that covered the cases when the app had not been opened for a while and there's slow or no network connectivity. I am pretty sure right now, it will try to update the caches and it will fail if there's no connectivity. Is that what we want?